TCIABrowser/TCIABrowser.py

```diff
@@ -1410,54 +1453,64 @@ def testBrowserDownloadAndLoad(self):
     widget.getCollectionValues()
     browserWindow = widget.browserWidget
     collectionsCombobox = browserWindow.findChildren('QComboBox')[0]
-    print 'Number of collections: ',collectionsCombobox.count
-    if collectionsCombobox.count> 0:
-      collectionsCombobox.setCurrentIndex(randint(0,collectionsCombobox.count-1))
+    print
+    'Number of collections: ', collectionsCombobox.count
```

Comment:
I see those were there in the previous code too. These newlines will result in print statement being empty, and second line treated as a comment.